deploy/addon/pulsar.md

```diff
@@ -0,0 +1,413 @@
+---
+title: Pulsar
+shortdesc: A distributed messaging and streaming platform on the publish/subsribe model
+tags:
+- addons
+keywords:
+- pulsar
+- messaging
+- cold storage
+- queue
+- hot storage
+- biscuit
+- rabbitmq
+- kafka
+---
+
+## Overview
+
+[Pulsar](https://pulsar.apache.org/) works on a publisher/subscriber model. It allows services to communicate asynchronously, with latencies ranging around 100 milliseconds. It is used for streaming analytics and data integration pipelines to ingest and distribute data. It is equally effective as messaging-oriented middleware for service integration or as a queue to parallelize tasks. It also enables you to create systems of event producers and consumers. Publishers communicate with subscribers asynchronously by broadcasting events.
+
+```
+client               Pulsar             client
+publishes to   =>    topic      <=  subscribes to
+
+"producer"                           "consumer"
+```
+
+They are several modes of subscription. A consumer may subscribe exclusively, or share the subscription with other consumers. There is the subscription mode types:
+
+- Exclusive (only one consumer for the subscription)
+- Failover (if a consumer fails, another one receives the message)
+- Shared (messages are distributed to several consumers)
+- Key_Shared (messages come with keys and go to consumers with the corresponding key)
+
+More on this in the [official documentation](https://pulsar.apache.org/docs/en/concepts-messaging/#subscriptions).
+
+A topic is defined this way:
+
+`{persistent|non-persistent}://tenant/namespace/topic`
+
+Tenants and namespaces allow for grouping and subgrouping of topics.
+
+A Clever Cloud Pulsar add-on is basically a immutable `tenant/namespace` where the tenant is your user id, and the namespace is the add-on id.
+It allows you to create and use topics following this pattern:
+
+`{persistent|non-persistent}://<CLEVERCLOUD_TENANT_ID>/<ADDON_ID>/<TOPIC_NAME>`
+
+## Version
+
+We maintain up-to-date Pulsar clusters based on the official Apache Pulsar release process. Your Pulsar add-on version is available in your add-on dashboard. Updates will be notified using Clever Cloud weekly updates on the [blog](https://www.clever-cloud.com/blog/).
+
+## Common use cases
```

Comment:
Almost of the items of the list use a different "example" pattern. Maybe make all the elements like:
`- <example>: explain it`

Reply:
Well no, it is too complicated to make generic as some are parts of other etc.